Makefile

```
@@ -1,6 +1,13 @@
 .PHONY: test
 
-DOCKER_RUN = docker-compose run --rm php
+DOCKER_COMPOSE = docker compose
+DOCKER_RUN = $(DOCKER_COMPOSE) run --rm php
+
+install:
+	$(DOCKER_COMPOSE) build --no-cache
```

```suggestion
	$(DOCKER_COMPOSE) build --no-cache
	${MAKE} vendor -B
```
If you want a real install.